<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatwindowBehaviour_Config</class>
 <widget class="QWidget" name="ChatwindowBehaviour_Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>534</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="kcfg_Beep">
        <property name="whatsThis">
         <string>Sounds the system bell when you receive an ASCII BEL (0x07) control character</string>
        </property>
        <property name="text">
         <string>Enable s&amp;ystem bell on incoming ASCII BEL</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="kcfg_HideUnimportantEvents">
        <property name="text">
         <string>Hide &amp;Join/Part/Nick events</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="kcfg_DisableExpansion">
        <property name="whatsThis">
         <string>Do not expand variables when sending text to the server. A variable starts with '%'; for example, %B will expand to the characters needed to make a text bold.</string>
        </property>
        <property name="text">
         <string>Disable variable (e.g. %C, %B, %G...) e&amp;xpansion</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QCheckBox" name="kcfg_RedirectServerAndAppMsgToStatusPane">
        <property name="text">
         <string>&amp;Redirect status messages to the server status window</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QCheckBox" name="kcfg_UseLiteralModes">
        <property name="whatsThis">
         <string>Keep channel mode string as a combination of characters instead of translating them into human readable words. E.g. '*** Channel modes: no messages from outside' will become '*** Channel modes: n'</string>
        </property>
        <property name="text">
         <string>Show raw mode characters (e.g. +o) instead of descriptions</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="KPluralHandlingSpinBox" name="kcfg_ScrollbackMax">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>How many lines to keep in buffers, 0 = unlimited</string>
        </property>
        <property name="specialValueText">
         <string>Unlimited</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="whatsThis">
         <string>How many lines to keep in buffers, 0 = unlimited</string>
        </property>
        <property name="text">
         <string>Scrollbac&amp;k limit:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kcfg_ScrollbackMax</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="indentHideUnimportantEventsExcludeActiveSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_HideUnimportantEventsExcludeActive">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="whatsThis">
           <string>If checked, only events concerning active and watched nicks will be displayed.</string>
          </property>
          <property name="text">
           <string>Except for watched nicks and nicks active within the</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="kcfg_HideUnimportantEventsExcludeActiveThreshold">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string comment="Is prefaced with: Except for watched nicks and nicks active within the">last 10 min</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string comment="Is prefaced with: Except for watched nicks and nicks active within the">last hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string comment="Is prefaced with: Except for watched nicks and nicks active within the">last day</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string comment="Is prefaced with: Except for watched nicks and nicks active within the">last week</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox3_2">
     <property name="title">
      <string>Marker Lines</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_AutomaticRememberLine">
        <property name="whatsThis">
         <string>Inserts a remember line into the chat window when you switch to another chat window or minimize the application.</string>
        </property>
        <property name="text">
         <string>Mark the last position in a chat window when it is hidden</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="indentAutomaticRememberLineOnlyOnTextChangeSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_AutomaticRememberLineOnlyOnTextChange">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Move the line only when new text is about to be shown</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_MarkerLineInAllViews">
        <property name="text">
         <string>Show manually inserted lines in all chat windows</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="kcfg_AutoWhoContinuousEnabled">
     <property name="title">
      <string>Enable Automatic User Information Look Up (/WHO)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>&amp;Max. number of users in a channel:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kcfg_AutoWhoNicksLimit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KPluralHandlingSpinBox" name="kcfg_AutoWhoNicksLimit">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="whoIntervalLbl">
        <property name="text">
         <string>Update inter&amp;val:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kcfg_AutoWhoContinuousInterval</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KPluralHandlingSpinBox" name="kcfg_AutoWhoContinuousInterval">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>30</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>41</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kpluralhandlingspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_Beep</tabstop>
  <tabstop>kcfg_HideUnimportantEvents</tabstop>
  <tabstop>kcfg_HideUnimportantEventsExcludeActive</tabstop>
  <tabstop>kcfg_HideUnimportantEventsExcludeActiveThreshold</tabstop>
  <tabstop>kcfg_DisableExpansion</tabstop>
  <tabstop>kcfg_RedirectServerAndAppMsgToStatusPane</tabstop>
  <tabstop>kcfg_UseLiteralModes</tabstop>
  <tabstop>kcfg_ScrollbackMax</tabstop>
  <tabstop>kcfg_AutomaticRememberLine</tabstop>
  <tabstop>kcfg_AutomaticRememberLineOnlyOnTextChange</tabstop>
  <tabstop>kcfg_MarkerLineInAllViews</tabstop>
  <tabstop>kcfg_AutoWhoContinuousEnabled</tabstop>
  <tabstop>kcfg_AutoWhoNicksLimit</tabstop>
  <tabstop>kcfg_AutoWhoContinuousInterval</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_AutoWhoContinuousEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_AutoWhoContinuousInterval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_AutomaticRememberLine</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_AutomaticRememberLineOnlyOnTextChange</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_HideUnimportantEvents</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_HideUnimportantEventsExcludeActive</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_HideUnimportantEvents</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_HideUnimportantEventsExcludeActiveThreshold</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
